provider/provider.go

```diff
@@ -28,6 +28,7 @@ func init() {
 		pt.K8sOffline:       k8sOfflineStoreFactory,
 		pt.BlobOnline:       blobOnlineStoreFactory,
 		pt.MongoDBOnline:    mongoOnlineStoreFactory,
+		pt.UNIT_TEST:        unitTestStoreFactory,
```

Comment:
Need more time to look through this. We do have a localonline so just want to make sure we're not overlapped